jasiekg25/foodsharing
# Feature/foodsharin 47 filtering by tags endpoint

- add endpoint for filtering by tags
- in search page show only offers which are not mine
- modify some mock-data

how to test:
- rebuild image and container
- log in Jan@Gargas.com test account and look at Offers. There should be 3 offers
- with postman you can check 
   get http://localhost:5001/offers?tags_ids=34&with_my=True
   
   expected 1 offer ("salatka")
